galaxy/values.yaml

```diff
@@ -512,6 +514,7 @@ extraFileMappings:
     useSecret: false
     applyToJob: false
     applyToWeb: true
+    applyToWorkflow: true
```

Comment:
I would default this to `false` since it only matters in the web handler

Reply:
I guess I forgot to change it back after testing.  Set the default back to `false` now.